modules/preloader/popupWindows.js

```diff
@@ -17,8 +18,6 @@ const web3Admin = require('../web3Admin.js');
 
 require('./include/setBasePath')('interface');
 
-// register with window manager
-ipc.send('backendAction_setWindowId');
```

Comment:
Why are these calls being removed? These are essential for hooking up the window id (and does event sending) correctly. Also, noticeably you didn't remove the event handler for this event from the Windows manager.